src/styles/block/card.scss

```
@@ -0,0 +1,78 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__image {
+    margin: 0 0 0 3px;
+    padding-bottom: 36px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0;
+    padding-bottom: 4px;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 0;
+    padding-bottom: 16px;
+    color: $secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    margin: 0;
+    display: flex;
+    justify-content: space-between;
+    padding-top: 12px;
+
+    &--title {
+      color: $secondary;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &--price {
+      color: $main-accent;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    margin: 16px 0 0;
+    width: 166px;
```

```suggestion
    width: 100%;
```
Or you can use min-width with static value here